src/HW6.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+public class HW6 {
+    public static void main(String[] args) {
+        // task 1
+        int[] array1 = new int[]{5, 7, 11, 20};
+        Arrays.sort(array1);
```

Comment:
It's not needed action, doesn't have any impact
Also it's a good idea to create a new method for it and then use the method in the main method